doc/source/examples/dem-mp/heated-packed-bed/heated-packed-bed.rst

```diff
@@ -0,0 +1,372 @@
+
+.. code-block:: text
+  :class: copy-button
+
+  mpirun -np 4 lethe-particles load-packed-bed.prm
+
+Then, the direction of gravity is changed with:
+
+.. code-block:: text
+  :class: copy-button
+
+  mpirun -np 4 lethe-particles flip-packed-bed.prm
+
+Finally, we run the simulation to heat the particles:
+
+.. code-block:: text
+  :class: copy-button
+
+  mpirun -np 4 lethe-particles heat-packed-bed.prm
+
+.. note::
+  In this example, the three stages require respectively around 16 minutes, 2 minutes and 2 minutes on 4 cores.
+
+
+---------------
+Post-processing
+---------------
+
+A Python post-processing code ``bed-postprocessing.py`` is provided with this example. It is used to compare the temperature of the packed-bed at three different heights :math:`h_1 = 4.0` cm, :math:`h_2 = 6.0` cm and :math:`h_3 = 7.3` cm (:math:`h = 0.0` cm corresponds to the top wall), with the results obtained experimentally and numerically by Beaulieu for stainless steel.
+
+.. figure:: images/heights.png
+    :height: 400
+    :align: center
+
+The post-processing code can be run with the following command. The argument is the folder which contains the ``.prm`` file.
+
+.. code-block:: text
+  :class: copy-button
+
+    python3 bed-postprocessing.py  --folder ./
+
+.. important::
+
+    You need to ensure that ``lethe_pyvista_tools`` is working on your machine. Click `here <../../../tools/postprocessing/postprocessing_pyvista.html>`_ for details.
+
+
+-------
+Results
+-------
+
+The following figure compares the temperature of the packed-bed at three different heights :math:`h_1 = 4.0` cm, :math:`h_2 = 6.0` cm and :math:`h_3 = 7.3` cm, with the results obtained by Beaulieu for stainless steel.
+
+.. figure:: images/mean-temperatures.png
+    :width: 500
+    :align: center
+
+The results show good agreement with the experimental and numerical results of Beaulieu but still undershoot a bit the temperature at :math:`h_1 = 4.0` cm compared to the experimental data. Our temperature curves are slightly higher than the ones numerically obtained by Beaulieu due to using the hertz contact radius instead of the analytical one. The higher Young's modulus also gives better results.
+
+It has been noticed while trying different methods to load the particles that the results change a lot according to the loading method but also depending on the walls friction coefficient.
+The following figure shows the results of the simulation with the ``friction coefficient wall`` set to :math:`1.0` instead of :math:`0.7`.
+
+.. figure:: images/mean-temperatures-friction.png
+    :width: 500
+    :align: center
+
+This ``friction coefficient wall`` allows to fit the experimental data better but it is debatable whether a friction coefficient of :math:`1.0` is realistic to model the Styrofoam walls used in the experiment. Also, other parameters like the ``rolling friction`` should have probably been adjusted as well to fit the experimental properties.
+
+---------
+Reference
```

Comment:
You could add a possibilities for extension and mention that there are other particles type and other results could be reproduced.